src/LightClient.ts

```diff
@@ -0,0 +1,25 @@
+export class LightClient {
+  shardName: string;
+
+  constructor(shardName: string)
+  {
+    this.shardName = shardName;
+  }
+
+  async getCurrentBlockHeight(): Promise<bigint> {
+    const {providers} = require("near-api-js");
+    const provider = new providers.JsonRpcProvider(
+      "https://rpc.testnet.near.org"
+    );
+
+    const rawResult = await provider.query({
+      request_type: "call_function",
```

Comment:
shouldn't this be view since its a non-mutating func?

Reply:
If `[this](https://docs.near.org/develop/integrate/rpc#view-methods)` documentation is up to date than this is the correct way to call functions. From my understanding this call does not refer to the difference between call and view functions in Near.